WireDomain/Sources/WireDomain/Event Processing/ConversationEventProcessor/ConversationMessageTimerUpdateEventProcessor.swift

```diff
@@ -26,15 +27,51 @@ protocol ConversationMessageTimerUpdateEventProcessorProtocol {
     ///
     /// - Parameter event: A conversation message timer update event.
 
-    func processEvent(_ event: ConversationMessageTimerUpdateEvent) async throws
+    func processEvent(_ event: ConversationMessageTimerUpdateEvent) async
 
 }
 
 struct ConversationMessageTimerUpdateEventProcessor: ConversationMessageTimerUpdateEventProcessorProtocol {
 
-    func processEvent(_: ConversationMessageTimerUpdateEvent) async throws {
-        // TODO: [WPB-10171]
-        assertionFailure("not implemented yet")
+    let userRepository: any UserRepositoryProtocol
+    let conversationRepository: any ConversationRepositoryProtocol
+    let conversationLocalStore: any ConversationLocalStoreProtocol
+    let messageRepository: any MessageRepositoryProtocol
+
+    func processEvent(_ event: ConversationMessageTimerUpdateEvent) async {
+        let userID = event.senderID
+        let conversationID = event.conversationID
+        let timer = Double(event.newTimer ?? 0)
+        let timestamp = event.timestamp
+
+        let conversation = await conversationRepository.fetchOrCreateConversation(
```

Comment:
I would expect that the local store is used here.

Reply:
removed since we're already using the conversation local store in the processor, we don't need the conversation repository, it is redundant. thx. fixed in https://github.com/wireapp/wire-ios/pull/2098/commits/25221ebfb5d7c61107b47436cd2fc321cb72924a